.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+on: [push]
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the source code
+      uses: actions/checkout@v2
+    - name: Validate SQF
+      run: python3 tools/sqf_validator.py
+    - name: Validate Config
+      run: python3 tools/config_style_checker.py
+    - name: Check for BOM
+      uses: arma-actions/bom-check@master
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the source code
+      uses: actions/checkout@master
+    - name: Lint (sqflint)
```

Comment:
Looking back from Achilles, we can remove this to save build times and massive logs because this isn’t really an improvement due to false positives.

Reply:
I'll remove it, I added it just to test this out.